R/ms_chart.R

```diff
@@ -57,13 +59,17 @@ assert_line <- function(data_y) {
 #' @examples
 #' library(officer)
 #' @example examples/02_linechart.R
-ms_linechart <- function(data, x, y, group = NULL, labels = NULL, asis = FALSE) {
+ms_linechart <- function(data, x, y, group = NULL, labels = NULL, asis = FALSE,
+                         secondary_x = FALSE, secondary_y = FALSE) {
```

Comment:
`secondary_x` is a second chart on the left y-axis. Got confused with the names.